eo-runtime/src/test/java/JulToSlf4jExtension.java

```diff
@@ -0,0 +1,24 @@
+import org.junit.jupiter.api.extension.BeforeAllCallback;
+import org.junit.jupiter.api.extension.ExtensionContext;
+import org.slf4j.bridge.SLF4JBridgeHandler;
+
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import static org.junit.jupiter.api.extension.ExtensionContext.Namespace.GLOBAL;
+
+public class JulToSlf4jExtension implements BeforeAllCallback {
+    private static boolean started = false;
+
+    /**
+     * Override the beforeAll() method
+     */
+    @Override
+    public void beforeAll(ExtensionContext context) {
+        if (!started) {
+            started = true;
+            context.getRoot().getStore(GLOBAL).put("JulToSlf4jExtension",this);
```

Comment:
@marceloamadeu you missed a space char after the comma

Reply:
@yegor256 has been added a space character after the comma.